resources/log4j2-test.xml

```diff
@@ -4,6 +4,9 @@
         <Console name="Console" target="SYSTEM_OUT">
             <PatternLayout pattern="[%-5p] %m%n" />
         </Console>
+        <File name="FileAppender" fileName="${sys:user.dir}/logs/cpg.log">

```

Comment:
does that write it to the current working directory, or the user's home dir?

Reply:
It writes to the current working directory. In an ideal case, I would've chosen the cpg/joern/ocular root directory as the target, but I'm not sure how that could be done. It may be possible if you create the appender in code instead of in the config file, but that feels a bit ugly.

Reply:
I think writing to the current working directory is fine when running tests (since those would most likely be run from the project directory in any case), but wouldn't work for general Joern/Ocular logs, for example, since those could be run from anywhere.

Reply:
CWD is great, let's keep it simple :)